src/cephadm/cephadm

```diff
@@ -565,6 +565,10 @@ def make_log_dir(fsid, uid=None, gid=None):
     makedirs(log_dir, uid, gid, LOG_DIR_MODE)
     return log_dir
 
+def make_var_run(fsid, uid, gid):
+    call_throws(['install', '-d', '-m0770', '-o', str(uid), '-g', str(gid),
```

Comment:
Use the same pattern as `make_log_dir` and `make_data_dir`?

```suggestion
    # type: (str, int, int) -> str
    if not uid or not gid:
        (uid, gid) = extract_uid_gid()
    run_dir = get_run_dir(fsid)
    makedirs(run_dir, uid, gid, RUN_DIR_MODE)
    return run_dir
```

Reply:
added the type line.  i think it's better to just require uid/gid tho since we don't need to magically extract it